data/api/server-server/joins-v2.yaml

```diff
@@ -207,9 +207,8 @@ paths:
                     title: SignedMembershipEvent
                     x-addedInMatrixVersion: "1.2"
                     description: |-
-                      Required if the `content` of the event in the request contained the `join_authorised_via_users_server`
-                        field. The signed copy of the membership event sent to other servers by the resident server,
-                        including the resident server's signature.
+                      The signed copy of the membership event sent to other servers by the resident server.
+                      Required if the event is signed by the resident server.
```

Comment:
Hrm. I think we need to clarify this a bit more. It just begs the question, "so when should the event be signed by the resident server".

Reply:
Would linking back to restricted joins be enough?

Reply:
Probably?

Reply:
I took another attempt at this, let me know what you think.